front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx

```diff
@@ -183,11 +183,21 @@ export default function AppRun({
               spec={spec}
               run={run}
               runRequested={false}
-              handleSetBlock={() => {}}
-              handleDeleteBlock={() => {}}
-              handleMoveBlockUp={() => {}}
-              handleMoveBlockDown={() => {}}
-              handleNewBlock={() => {}}
+              handleSetBlock={() => {
```

Comment:
Can't have empty code block AND can't have undefined or null functions here.

Reply:
I'd rather have a `@ts-expect-error` than this no?